src/_locales/en/messages.json

```diff
@@ -39,5 +42,36 @@
     },
     "volume_mute": {
         "message": "Volume: toggle mute"
+    },
+    "interact_dialog_text": {
```

Comment:
Totally agree re: the clutter, I was just trying to make it obvious why some commands are still allowed to execute while `play-pause`, `next` & `previous` are not. And I guess the user will realise pretty quick that the escape key doesn't close the dialog. I'd probably even leave out the "just this once" part as well.

Reply:
I used your css pretty-much as-is, although I switched the subtext colour for an opacity value in case Spotify invert their dialog colour scheme sometime in the future.

Reply:
Actually is it worth changing "Chrome" to be non-specific, as this extension should work with any chromium-based browser? I don't even use it with Chrome myself, other than for testing.

Reply:
In believe this is resolved.